test/Interpreter/struct_extra_inhabitants.swift

```diff
@@ -1,7 +1,7 @@
 // RUN: %empty-directory(%t)
 
 // -- build resilient library
-// RUN: %target-build-swift -force-single-frontend-invocation -Xfrontend -enable-resilience -module-name ExtraInhabitantResilientTypes -emit-module-path %t/ExtraInhabitantResilientTypes.swiftmodule -parse-as-library -c -o %t/ExtraInhabitantResilientTypes.o %S/Inputs/struct_extra_inhabitants_ExtraInhabitantResilientTypes.swift
+// RUN: %target-build-swift -force-single-frontend-invocation -enable-resilience -module-name ExtraInhabitantResilientTypes -emit-module-path %t/ExtraInhabitantResilientTypes.swiftmodule -parse-as-library -c -o %t/ExtraInhabitantResilientTypes.o %S/Inputs/struct_extra_inhabitants_ExtraInhabitantResilientTypes.swift
```

Comment:
Hey @jckarter, why are you passing -force-single-frontend-invocation here?

Reply:
Without it, the frontend appears to only emit the module, not both the module and .o file. I don't understand why. cc @jrose-apple

Reply:
I think the driver doesn't special-case single-file `-c` except under `-force-single-frontend-invocation`, and therefore expects the location of the output file to be controlled by an output file map (because it wouldn't work for two files). Mind filing a bug? (This is similar to [SR-7704](https://bugs.swift.org/browse/SR-7704) but easier to do something about.)

Reply:
Here you go: https://bugs.swift.org/browse/SR-10116